plugins/guests/astra/guest.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../linux/guest'
+
+module VagrantPlugins
+  module GuestAstra
+    class Guest < VagrantPlugins::GuestLinux::Guest
+      # Name used for guest detection
+      GUEST_DETECTION_NAME = "astra".freeze
```

Comment:
Is this all that's required with the Astra guest? Just a different name to check against inside the release file?

Reply:
Yes, it is debian based distro with custom DE (Fly). 
/etc/os-releses contains:
`  PRETTY_NAME="Astra Linux (Orel 2.12.14)"  
  NAME="Astra Linux (Orel)"  
  ID=astra  
  ID_LIKE=debian  
`

Reply:
Hmm ok...I think in this case our linux guest detection for for `/etc/os-release` file is a little off.

https://github.com/hashicorp/vagrant/blob/1dda6cc14c2d221a8e2e7b5091fed2fca9ba1ecf/plugins/guests/linux/guest.rb#L9-L11

It looks at `ID`, but in this case we could also try to match against `ID_LIKE` since it seems that Astra is the same as Debian. We could potentially add a fourth check there at the end as a failsafe if nothing else matched. That would save us adding an additional guest capability here.